apps/01PigLatin.js

```diff
@@ -50,42 +50,42 @@ function pigLatin(word) {
 
 
   if (vowelIndex === 0) {
-      return word + 'yay';
-    } else {
-      return restWord + firstPart + 'ay';
-    }
+    return word + 'yay';
```

Comment:
I totally thought this part was supposed to also be 'ay', but looks like you got it from the beginning. *thumbsup* Good job on reading directions. :)